plugins/processors/starlark/starlark.go

```diff
@@ -40,6 +41,46 @@ type Starlark struct {
 	applyFunc *starlark.Function
 	args      starlark.Tuple
 	results   []telegraf.Metric
+	cache     Cache
+}
+
+// The struct that defines the shared cache used
+type Cache struct {
+	content map[string]interface{}
+	mutex   sync.RWMutex
```

Comment:
I don't think the mutex is necessary, as this belongs to the plugin and is never called in parallel.

Reply:
@ssoroka ok